app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <% if Company.find_by(name: '3sixD').logo.present? %>
+      <% @default_company = Company.find_by(name: '3sixD') || Company.find_by(name: '3sixd') %>
```

Comment:
@godfreym you are using `||` operator to assign the `@default_company` instance. is this a ruby short cut because `||` is supposed to deal with `truthy of falsy ` values.

I think what is more logical is to use two if structures for assigning the default company

if '3sixD' is there then default company is that

if 3sixd' is there then the default company is that